libs/MailDB/src/PgMailDB.cpp

```diff
@@ -279,74 +242,6 @@ void PgMailDB::InsertEmail(const std::string_view sender, const std::vector<std:
     }
 }
 
-std::vector<Mail> PgMailDB::RetrieveEmails(const std::string_view user_name, bool should_retrieve_all) const
-{
-    pqxx::nontransaction ntx(*m_conn);
-
-    uint32_t user_id = RetriveUserId(user_name, ntx);
-
-    std::string additional_condition = "";
```

Comment:
std::string additional_condition = ""; here is a redudant string initialization

Reply:
This has been implemented by @zikkend, please leave feedback in his pull-request about related issue